projects/ccf-rui/src/app/shared/components/slices-input/slices-input.component.html

```diff
@@ -15,13 +15,13 @@
   <mat-form-field class="field">
     <mat-label class="text label">Thickness</mat-label>
     <input matInput ccfNumbersOnly class="input" type="number" [value]="slicesConfig.thickness"
-           (input)="updateSlicesData($event, 'thickness')"  matTooltip="Enter thickness of tissue slices">
+           (input)="updateSlicesData($event, 'thickness')"  matTooltip="Enter thickness of tissue sections">
     <div matSuffix class="suffix" [class.show]="hasThicknessValue">μm</div>
   </mat-form-field>
 
   <mat-form-field class="field">
     <mat-label class="text label"># Slices</mat-label>
```

Comment:
Forgot one :stuck_out_tongue:

Reply:
Thanks, fixed in new commit https://github.com/hubmapconsortium/ccf-ui/pull/746/commits/afb4290c9806bff142c93174c60d209f7bd3666e.